save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/StringUtils.kt

```diff
@@ -0,0 +1,16 @@
+@file:Suppress("HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE")
+
+package com.saveourtool.save.utils
+
+/**
+ * @return short [String] for list values
+ */
+fun List<String>?.listToShortString(): String {
+    this?.run {
+        return if (size <= 2) {
+            this.joinToString(" ,")
+        } else {
+            "${first()} ... ${last()}"
+        }
+    } ?: return ""
```

Comment:
Please lift the `return` out of the `run()` body, and change:

```suggestion
    }.orEmpty()
```

The whole function can be converted to an expression function then.

Reply:
Fixed